frontends/web/src/components/rates/rates.tsx

```diff
@@ -99,6 +99,9 @@ interface ProvidedProps {
     unstyled?: boolean;
     skipUnit?: boolean;
     noAction?: boolean;
+    typeClassName?: string;
+    sign?: string | undefined;
```

Comment:
nit: `?` indicates optional and should already allow `undefined `

Reply:
Thx, updated!